common/changes/@rushstack/heft-node-rig/user-danade-HeftActionAlias_2023-06-06-23-16.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-node-rig",
+      "comment": "Add \"heft start\" alias that maps to \"heft build-watch --serve\"",
+      "type": "patch"
```

Comment:
For semver this might actually rate as a minor in heft-node-rig, since it has released 1.0 already.

Reply:
Good point. Will update